src/environments/environment.electron.prod.ts

```diff
@@ -2,5 +2,7 @@ export const environment = {
   solarputty_download_url: '',
   production: true,
   electron: true,
-  current_version:'v3'
+  current_version:'v3',
+  compute_id:'local',
+  vm_compute_id:'vm'
```

Comment:
`vm_compute_id` should be removed (and below), is will be deprecated in version 3.0. See https://github.com/GNS3/gns3-web-ui/issues/1161